packages/manager/modules/telecom-spare/package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "@ovh-ux/manager-telecom-spare",
+  "version": "0.0.0",
+  "private": true,
+  "license": "BSD-3-Clause",
+  "files": [
+    "dist"
+  ],
+  "main": "./dist/cjs/index.js",
```

Comment:
in order to have better performance we are avoiding compiling modules with rollup now, we prefer webpack to do the job for us (please look for example https://github.com/ovh-ux/manager/blob/master/packages/manager/modules/sms/package.json). Basically you just have to remove scripts and modify the main target